scaleway/resource_k8s_cluster.go

```diff
@@ -3,6 +3,7 @@ package scaleway
 import (
 	"context"
 	"fmt"
+	"slices"
```

Comment:
We should bump to go 1.21

Reply:
ok I replaced the call to slices by a simple if as there is only two elements